db/migrate/20170828021241_user_schema_rename_changes.rb

```diff
@@ -0,0 +1,6 @@
+class UserSchemaRenameChanges < ActiveRecord::Migration[5.1]
```

Comment:
migration names should follow a standard descriptive name. This is too generic

Reply:
I'd probably make two migrations:
ChangeNameToFirstName
ChangeNicknameToUsername